pwndbg/commands/elf.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import gdb
+import pwndbg.commands
+from elftools.elf.elffile import ELFFile
+
+@pwndbg.commands.Command
+def elfheader():
+    """
+    Prints the section mappings contained in the ELF header.
+    """
+    exe = str(pwndbg.auxv.get()['AT_EXECFN'])
+    with open(exe, 'rb') as f:
+        elffile = ELFFile(f)
+        load_segment = elffile.get_segment(3)
+        segment_base = load_segment['p_vaddr']
+        for section in elffile.iter_sections():
```

Comment:
Can we print these in linear order? I'm not sure they're guaranteed to be, by default